app/controllers/user_my_modules_controller.rb

```diff
@@ -13,7 +13,9 @@ def index
         render json: {
           html: render_to_string(
             partial: 'index.html.erb'
-          )
+          ),
+          my_module_id: @my_module.id,
```

Comment:
Same here, add comment `# Used for counter badge` or something along those lines.